### PR TITLE
fix telegraf rs false errors

### PR DESCRIPTION
**Pull request by pfrcks**
When configmap is applied to the cluster TELEMETRY_RS_TELEGRAF_DISABLED true was not being written to the /opt/env_vars file which is used by liveness probe to get the env variable settings

### CODE REVIEW

File: .trivyignore [hunk 1]
```diff
@@ -4,6 +4,10 @@ GHSA-fr2g-9hjm-wr23
...
 CVE-2023-3978
 CVE-2023-44487
+GHSA-jq35-85cj-fj4p
+
+#telegraf HIGH
+GHSA-m425-mq94-257g
 
 # ruby HIGH
-CVE-2017-10784
+CVE-2017-10784
```
Comment (ganga1980) [inline .trivyignore]: Does this warrant for us to upgrade ruby?

Comment (pfrcks) [reply .trivyignore]: dont know this is the same vuln we have had for some time now

Comment (pfrcks) [reply .trivyignore]: https://github.com/microsoft/Docker-Provider/commit/d5f9bb7c5bec6a9b34d3b123726235f242026ec4#diff-a2b39041fe7520e8037b4e596d015ea4825043e383f3fa5f85d5ac7f06f4c177

Comment (jatakiajanvi12) [reply .trivyignore]: This doesn't need ruby update. This will need fluentd upgrade because it is a dependency of fluentd.

Comment (ganga1980) [reply .trivyignore]: Lets discuss about this in todays standup.   I think, this change can be merged.

**Review by ganga1980 — APPROVED** [review]

**Pull request merged**